### DISCUSSION TITLE
关于企业版的bing

### DISCUSSION THREAD

**Discussion by q2408808** [viopsa233/AI-testers-Forum#94] [交流]
![1689817283462](https://github.com/viopsa233/AI-testers-Forum/assets/128444814/e69080a5-e8a9-440b-bf42-8cce7a9ca196)
交流下来感觉还行。不过很久没用必应聊天了，不确定跟普通必应有什么区别，你们有用过的吗？

**Comment by twsec-pro** [comment]
怎么搞的？

**Reply by q2408808** [reply]
有365权限就行
